src/features/Content/MoviesList/Tiles/OneMovie/index.js

```diff
@@ -17,41 +18,38 @@ export function OneMovie() {
         axios.get(`https://api.themoviedb.org/3/movie/${id}?api_key=eb6efa05b2b8136a07d256a13fbb4f15&language=en-US`)
           .then(({ data }) => {
             setApi({ state: "success", data });
-            console.log(data)
+            // console.log(data)
           })
       } catch (error) {
         (setApi({ state: "error", }));
       }
     }
     setTimeout(axiosApi, 500)
   }, [id]);
-  console.log(api);
+  // console.log(api);
 
   return (
     <>
       {api.state === "loading"
-        ? (
-          <p>
-            <Loader />
-          </p>
-        )
+        ? (<Loader />)
```

Comment:
? (<Loader />)  to powinno tu być?

Reply:
pytasz o Loader? według mnie tak, bo zagrywam po to do osobnego api, które musi się wczytać